go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	golang.org/x/sys v0.0.0-20200806125547-5acd03effb82
 	k8s.io/api v0.18.6
 	k8s.io/apimachinery v0.18.6
+	k8s.io/client-go v0.18.6
```

Comment:
Do we need this? It looks like it's not imported anywhere.

Reply:
fixed